src/GraphQL.Tests/Serialization/SystemTextJson/InputsConverterTests.cs

```diff
@@ -0,0 +1,302 @@
+            object values = actual["values"];
+            values.ShouldBeAssignableTo<IEnumerable<object>>();
+        }
+
+        [Fact]
+        public void Deserialize_ComplexValue()
+        {
+            string json = @"
+                {
+                    ""complex"": {
+                        ""int"": 123,
+                        ""double"": 123.456,
+                        ""string"": ""string"",
+                        ""bool"": true
+                    }
+                }
+            ";
+
+            var actual = JsonSerializer.Deserialize<Inputs>(json, _options);
+
+            var complex = actual["complex"].ShouldBeAssignableTo<IDictionary<string, object>>();
+            complex["int"].ShouldBe(123);
+            complex["double"].ShouldBe(123.456);
+            complex["string"].ShouldBe("string");
+            complex["bool"].ShouldBe(true);
+        }
+
+        [Fact]
+        public void Deserialize_MixedValue()
+        {
+            string json = @"
+                {
+                    ""int"": 123,
+                    ""complex"": {
+                        ""int"": 123,
+                        ""double"": 123.456,
+                        ""string"": ""string"",
+                        ""bool"": true
+                    },
+                    ""bool"": true
+                }
+            ";
+
+            var actual = JsonSerializer.Deserialize<Inputs>(json, _options);
+
+            actual["int"].ShouldBe(123);
+            actual["bool"].ShouldBe(true);
+
+            var complex = actual["complex"].ShouldBeAssignableTo<IDictionary<string, object>>();
+            complex["int"].ShouldBe(123);
+            complex["double"].ShouldBe(123.456);
+            complex["string"].ShouldBe("string");
+            complex["bool"].ShouldBe(true);
+        }
+
+        [Fact]
+        public void Deserialize_Nested_SimpleValues()
+        {
+            string json = @"
+                {
+                    ""value1"": ""string"",
+                    ""dictionary"": {
+                        ""int"": 123,
+                        ""double"": 123.456,
+                        ""string"": ""string"",
+                        ""bool"": true
+                    },
+                    ""value2"": 123
+                }
+            ";
+
+            var actual = JsonSerializer.Deserialize<Nested>(json, _options);
+
+            actual.Value1.ShouldBe("string");
+            actual.Value2.ShouldBe(123);
+        }
+
+        [Fact]
+        public void Serialize_SimpleValues()
+        {
+            var source = new Nested
+            {
+                Value2 = 123,
+                Value1 = null
+            };
+
+            string json = JsonSerializer.Serialize(source, _options);
+
+            json.ShouldBeCrossPlatJson(
+                @"{
+  ""value1"": null,
+  ""dictionary"": null,
+  ""value2"": 123
+}".Trim());
+        }
+
+        [Fact]
+        public void Serialize_Nested_SimpleValues()
```

Comment:
Serialization of `Inputs` objects is now supported, so these 3 tests are enabled.  That's the only notable difference made to this file compared to `ObjectDictionaryConverterTests`.